packages/time/__tests__/semester.test.js

```diff
@@ -0,0 +1,59 @@
+// @ts-check
+
+import {expect} from 'chai';
+import Semester from '../lib/semester.js';
+
+const verifyReadable = (semesterString, year, season) => {
+	const result = Semester.parse(semesterString);
+	expect(result).to.not.be.an('boolean');
+	if (result) {
+		expect(result.year).to.equal(year);
+		expect(result.season).to.equal(season);
+	}
+};
+
+const verifyString = (semesterString, readableString, emoji = false) => {
+	const result = new Semester(semesterString);
+	expect(result).to.not.be.an('boolean');
+	if (result) {
+		expect(result.toString(emoji)).to.equal(readableString);
+	}
+};
+
+describe('Unit > Semester', function () {
+	it('constructor', function () {
+		const blankSemester = new Semester('');
+		expect(blankSemester.raw).to.be.empty;
+		expect(blankSemester.year).to.be.equal(-1);
+		expect(blankSemester.toString()).to.be.equal('Semesters');
+
+		expect(function () {
+			// eslint-disable-next-line no-new
+			new Semester('2034FALL');
+		}).to.throw('Semester "2034FALL" is not a valid semester');
```

Comment:
The pattern I use is

```suggestion
		try {
			const semester = new Semester('2034FALL');
			expect(semester).to.equal('should have thrown');
		} catch (error) {
			expect(error.message).to.equal('Semester "2034FALL" is not a valid semester');
		}
```